__tests__/queryUtils.test.js

```diff
@@ -0,0 +1,39 @@
+const db = require('../db/connection.js');
+const request = require('supertest');
+const app = require('../app.js');
+
+describe('checkArticleExists', () => {
+    test('GET request returns article object if article exists', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(( { body }) => {
+            expect(body.article).toMatchObject({
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: expect.any(String),
+                votes: 100,
+            })
+        });
+    });
```

Comment:
I love the idea of testing your utils functions!!
However, does this function just behave in the same way as your `selectArticleByArticleId` function?